lib/connection.ts

```diff
@@ -237,9 +237,6 @@ export class Connection extends Entity {
     } else {
       let connectionOptions = options as ConnectionOptions;
       if (!connectionOptions) connectionOptions = { transport: "tls" };
-      if (connectionOptions.operationTimeoutInSeconds == undefined) {
-        connectionOptions.operationTimeoutInSeconds = defaultOperationTimeoutInSeconds;
```

Comment:
doesn't line 247 be impacted by this, by not having the default value in the `connectionOptions`?

Reply:
Good point! We want to pass `operationTimeoutInSeconds` to `create_connection` too.

Reply:
Looks that `rhea` doesn't do anything with `operationTimeoutInSeconds` in `create_connection`, but to keep the original behavior I will add it back.